public/components/notebooks/docs/dev/Zeppelin_backend_adaptor.md

```diff
@@ -75,7 +75,7 @@
 
       - Change transport.type to http
       - host → localhost (if running on same machine as Zeppelin) & port → 9200
-      - username: admin & password: admin
+      - username: `admin` & password: `<admin password>`
```

Comment:
If we're not going to provide specific values it can be better to not use a placeholder with brackets. Maybe "Use the username `admin` and the configured password (default `admin`)".